products/cpdb/projects/metadata.yml

```diff
@@ -0,0 +1,393 @@
+          spent_nccstate: {name: spnccstate}
+          spent_nccfederal: {name: spnccfed}
+          spent_nccother: {name: spnccother}
+          spent_noncitycost: {name: spncc}
+          spent_total: {name: sptotal}
+          geometry: {name: geometry}
+
+    - name: projects_polygons
+      type: shapefile
+      filename: cpdb_projects_poly.shp.zip
+      overrides:
+        display_name: Capital Projects Database (CPDB) - Projects (Polygons)
+        columns:
+          maprojid: {name: maprojid}
+          magency: {name: magency}
+          projectid: {name: projectid}
+          description: {name: descript}
+          plannedcommit_ccnonexempt: {name: pcccnonex}
+          plannedcommit_ccexempt: {name: pcccex}
+          plannedcommit_citycost: {name: pccc}
+          plannedcommit_nccstate: {name: pcnccstate}
+          plannedcommit_nccfederal: {name: pcnccfed}
+          plannedcommit_nccother: {name: pcnccother}
+          plannedcommit_noncitycost: {name: pcncc}
+          plannedcommit_total: {name: pctotal}
+          spent_total_checkbooknyc: {name: sptotalcb}
+          maxdate: {name: maxdate}
+          mindate: {name: mindate}
+          magencyacro: {name: magenacro}
+          magencyname: {name: magenname}
+          typecategory: {name: typecat}
+          ccpversion: {name: ccpversion}
+          adopt_ccnonexempt: {name: adccnonex}
+          adopt_ccexempt: {name: adccex}
+          adopt_citycost: {name: adcc}
+          adopt_nccstate: {name: adnccstate}
+          adopt_nccfederal: {name: adnccfed}
+          adopt_nccother: {name: adnccother}
+          adopt_noncitycost: {name: adncc}
+          adopt_total: {name: adtotal}
+          allocate_ccnonexempt: {name: alccnonex}
+          allocate_ccexempt: {name: alccex}
+          allocate_citycost: {name: alcc}
+          allocate_nccstate: {name: alnccstate}
+          allocate_nccfederal: {name: alnccfed}
+          allocate_nccother: {name: alnccother}
+          allocate_noncitycost: {name: alncc}
+          allocate_total: {name: altotal}
+          commit_ccnonexempt: {name: coccnonex}
+          commit_ccexempt: {name: coccex}
+          commit_citycost: {name: cocc}
+          commit_nccstate: {name: conccstate}
+          commit_nccfederal: {name: conccfed}
+          commit_nccother: {name: conccother}
+          commit_noncitycost: {name: concc}
+          commit_total: {name: cototal}
+          spent_ccnonexempt: {name: spccnonex}
+          spent_ccexempt: {name: spccex}
+          spent_citycost: {name: spcc}
+          spent_nccstate: {name: spnccstate}
+          spent_nccfederal: {name: spnccfed}
+          spent_nccother: {name: spnccother}
+          spent_noncitycost: {name: spncc}
+          spent_total: {name: sptotal}
+          geometry: {name: geometry}
+
+  attachments:
+    - cpdb_data_dictionary.xlsx
+    - source_data_versions.csv
+
+columns:
+  - name: ccpversion
+    display_name: ccpversion
```

Comment:
So with the `display_name`s, do these match what was on Socrata before we pushed? IIRC, the display_name is the column header that Socrata spits out for exported files, e.g. csv's